.custom_wordlist.txt

```diff
@@ -102,6 +102,7 @@ https
 iharness
 IMailDelivery
 IMailer
+InformationInfrastructure
```

Comment:
I am not sure if it is the correct approach to add these compound words from URLs into this list.